app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+class ArticlesController < ApplicationController
```

Comment:
__[Reek]__

[InstanceVariableAssumption] ArticlesController assumes too much for instance variable '@article'


<!-- Comment by [SideCI](https://sideci.com) -->